src/truststore/_windows.py

```diff
@@ -400,12 +418,31 @@ def _get_and_verify_cert_chain(
         )
 
         # Check status
-        # TODO: Better error messages
         error_code = policy_status.dwError
         if error_code:
-            err = ssl.SSLCertVerificationError(
-                f"Certificate chain policy error {error_code:#x} [{policy_status.lElementIndex}]"
+
+            # Try getting a human readable message for an error code.
```

Comment:
Instead of doing this with FormatMessageW I think you could raise https://docs.python.org/3/library/ctypes.html#ctypes.WinError to have ctypes take care of formatting the message, then catch it to extract the message and reraise as SSLCertVerificationError

Reply:
This is a good idea, I'll try this once I land.